client/components/checkout.js

```diff
@@ -33,14 +34,20 @@ class Checkout extends Component {
   render() {
     const {sumAmount} = this
     const totalAmount = sumAmount(this.props.cart.orderProducts)
+    const userId = this.props.cart.userId
+    const checkoutCart = this.props.checkoutCart
     const order = {
       totalAmount: totalAmount
     }
 
     return (
       <div className="containerCheckout">
         <div className="totalAmtCheckout">{`Your Order Total $${totalAmount}`}</div>
-        <StripeCheckoutButton order={order} />
+        <StripeCheckoutButton
+          order={order}
+          checkoutCart={checkoutCart}
```

Comment:
checkoutCart = {cartCheckout}. If we make that updated change above.

Reply:
yes